src/test/java/com/github/wasiqb/coteafs/datasource/data/XmlData.java

```diff
@@ -0,0 +1,19 @@
+package com.github.wasiqb.coteafs.datasource.data;
+
+import com.github.wasiqb.coteafs.datasource.annotation.DataFile;
+import lombok.Data;
+
+/**
+ * @author Faisal Khatri
+ * @since Aug 31, 2020
+ *
+ */
+@Data
+@DataFile ( folderPath = "src\\test\\resources\\files\\xmlfiles", fileName = "testfile.xml")
```

Comment:
Same change as mentioned in previous comment.

Reply:
Done